diff --git a/lua/luasnip/nodes/snippet.lua b/lua/luasnip/nodes/snippet.lua
index 9d2165a..25a1527 100644
--- a/lua/luasnip/nodes/snippet.lua
+++ b/lua/luasnip/nodes/snippet.lua
@@ -16,6 +16,7 @@ local extend_decorator = require("luasnip.util.extend_decorator")
 local source = require("luasnip.session.snippet_collection.source")
 local loader_util = require("luasnip.loaders.util")
 local trig_engines = require("luasnip.nodes.util.trig_engines")
+local log = require("mc.util.vlog")
 
 local true_func = function()
 	return true
@@ -626,13 +627,14 @@ end
 
 -- returns copy of snip if it matches, nil if not.
 function Snippet:matches(line_to_cursor)
+	log.info("matches")
 	local match, captures = self.trig_matcher(line_to_cursor, self.trigger)
 
 	-- Trigger or regex didn't match.
 	if not match then
 		return nil
 	end
-
+	log.info("try expand param")
 	local expand_params =
 		self:resolveExpandParams(line_to_cursor, match, captures)
 	if not expand_params then
@@ -1230,6 +1232,7 @@ function Snippet:subtree_set_pos_rgrav(pos, direction, rgrav)
 	end
 	adjust_children_rgravs(self, pos, child_from_indx, direction, rgrav)
 end
+
 -- changes rgrav of all nodes and all endpoints in this snippetNode to `rgrav`.
 function Snippet:subtree_set_rgrav(rgrav)
 	self.mark:set_rgravs(rgrav, rgrav)
